server/plugin/command_test.go

```diff
@@ -477,6 +505,68 @@ func TestHandleSearchAndShare(t *testing.T) {
 	}
 }
 
+func TestHandleCreateIncident(t *testing.T) {
+	p := Plugin{}
+	mockAPI := &plugintest.API{}
+	args := &model.CommandArgs{
+		UserId: testutils.GetID(),
+	}
+	for _, testCase := range []struct {
```

Comment:
dont use loop if there is only 1 subtest

Reply:
this test case is removed now